checkstyle/checkstyle.xml

```diff
@@ -269,12 +269,6 @@
     </module>
     <module name="OverloadMethodsDeclarationOrder"/>
     <module name="VariableDeclarationUsageDistance"/>
-    <module name="CustomImportOrder">
-      <property name="sortImportsInGroupAlphabetically" value="true"/>
-      <property name="separateLineBetweenGroups" value="true"/>
-      <property name="customImportOrderRules" value="STATIC###THIRD_PARTY_PACKAGE"/>
-      <property name="tokens" value="IMPORT, STATIC_IMPORT, PACKAGE_DEF"/>
-    </module>
```

Comment:
removing import orders check style configuration. 
Reason: 
 1. It is taking extra effort to fix every single file import of orders
 2. Import of order is not big deal to understand code, what matter is code itself
 3. This checkstyle is not even compatible with official folio documentation
 4. In these days, it is better to trust IDEA, so more focus one code itself.
 
If you have strong reason to keep this, please give, or it will be removed.

Note: Only order of imports is being deleted. not deleting unused import check and wildcard style

Reply:
Please revert this change as it verifies the code style of the repository.

Reply:
it is not about code style, removing part is for import order style. Is there any reason to keep order of imports?

Reply:
We should follow order of imports that owners of this module define

Reply:
Reasons to have it:

- Merge Conflict Reduction: Consistent import ordering reduces the likelihood of merge conflicts. When multiple developers work on the same file, inconsistent import orders can lead to unnecessary conflicts that could have been avoided.

- Tool Compatibility: While modern IDEs like IntelliJ IDEA offer tools to manage imports automatically, not all team members might use the same IDE or have the same configuration. A standardized import order ensures consistency across different development environments.

- Code Quality and Maintenance: Maintaining a structured import order is part of adhering to coding standards that promote high-quality code. It ensures that the codebase remains maintainable and comprehensible, especially as the project grows.

Reply:
1. Solving Merge conflict for imports are way easier than fix imports one by one based on checkstyle.
2. I don't think many people who contribute this repo in this year, are not using Intelij IDEA. even if you use other tool still they can configure their ide instead of many intellij idea user configure.
3. :) 

@psmagin basically, are you totally okay, if someone come from other team and try to contribute and spend extra 10-20 minut to fix import checkstyle issue.

Why I am saying is that, I saw this same checkstyle in other module and spend extra time on to fix just import orders which turn out be different from other modules. We have similar two ticket to contribute this module, I am trying to avoid other people to spend useless time to fix import-order style. 

If you still thinking to revert this based on these informations, please update your checkstyle based on documentation, for us easier to use same imports style for all modules.
https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/1378531/Java+code+formatter+for+IDEs

Reply:
A small side note, you need to install CheckStyle-IDEA plugin into the IDEA first

Reply:
@psmagin thanks for sharing I will use definetly